imagetagger/imagetagger/images/views.py

```diff
@@ -392,10 +396,12 @@ def view_imageset(request, image_set_id):
     user_teams = Team.objects.filter(members=request.user)
     imageset_edit_form = ImageSetEditForm(instance=imageset)
     imageset_edit_form.fields['main_annotation_type'].queryset = AnnotationType.objects.filter(active=True)
+    imageset_description_md = bleach.clean(markdown.markdown(imageset.description), markdown_tags, markdown_attrs)
```

Comment:
This can be None:

```
'NoneType' object has no attribute 'strip'
```

Reply:
This makes creation of image sets currently impossible.

Reply:
closed with fe24276